### PR TITLE
Add http interceptor

### LINKED ISSUE #75: Create TokenInterceptor
At present the code [here](https://github.com/gothinkster/angular-realworld-example-app/blob/master/src/app/shared/services/api.service.ts) seems repetitive, imho an interceptor should be  perfect use case for this.

### PR DESCRIPTION
**Pull request by vivekimsit**
closes #75

### CODE REVIEW

File: src/app/shared/interceptors/http.token.interceptor.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+export class HttpTokenInterceptor implements HttpInterceptor {
+  constructor(private jwtService: JwtService) {}
+
+  intercept(req: HttpRequest<any>, next: HttpHandler): Observable<HttpEvent<any>> {
+    const headersConfig = {
+      'Content-Type': 'application/json',
+      'Accept': 'application/json'
+    };
+
+    if (this.jwtService.getToken()) {
```
Comment (Toxicable) [inline src/app/shared/interceptors/http.token.interceptor.ts]: no need to call `getToken` twice here, put the result in a local variable

File: package.json [hunk 2]
```diff
@@ -29,6 +29,7 @@
     "zone.js": "^0.8.14"
   },
   "devDependencies": {
+    "@angular-devkit/core": "0.0.29",
```
Comment (Toxicable) [inline package.json]: why do we need this?

Comment (vivekimsit) [reply package.json]: There is issue while doing `ng serve`, I managed to find better solution.

File: src/app/shared/interceptors/http.token.interceptor.ts (hunk 1 above)
Comment (vivekimsit) [reply src/app/shared/interceptors/http.token.interceptor.ts]: done

**Pull request merged**